apple/swift-docc
# Fix broken link and missing space in @Resources documentation

Bug/issue #, if applicable: rdar://102737063

## Summary

This fixes a broken link and a missing space on https://www.swift.org/documentation/docc/resources 

## Dependencies

None

## Testing

- Run `swift run docc preview Sources/docc/DocCDocumentation.docc`
- Navigate to http://localhost:8080/documentation/docc/resources and verify that the `Videos` link resolved and works.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
